package.json

```diff
@@ -12,75 +12,62 @@
   },
   "packageManager": "yarn@3.2.0",
   "dependencies": {
-    "@actions/core": "^1.4.0",
-    "@actions/github": "^5.0.0",
     "@angular-devkit/build-angular": "14.0.0-next.2",
     "@angular/benchpress": "0.3.0",
+    "@angular/compiler-cli": "14.0.0-next.3",
     "@babel/core": "^7.16.0",
-    "@bazel/bazelisk": "^1.10.1",
-    "@bazel/buildifier": "^4.0.1",
     "@bazel/concatjs": "4.6.0",
     "@bazel/esbuild": "4.6.0",
-    "@bazel/jasmine": "4.6.0",
     "@bazel/protractor": "4.6.0",
     "@bazel/runfiles": "4.6.0",
     "@bazel/terser": "4.6.0",
     "@bazel/typescript": "4.6.0",
+    "@google-cloud/storage": "^5.18.1",
```

Comment:
I realize the `update-script` folder in `bazel/browsers` is technically not shipped to the NPM artifact. It's an internal script for us, so we could move it to dev dependencies. Let's just do that as a follow-up though.